src/generate/formats/writer/yaml/utils.ts

```diff
@@ -236,6 +241,12 @@ export function createFlowInputsString(
       }
     } else if (input.type === "template") {
       currentInputString += `${convertTemplateInput(input.value as string, trigger, loop)},`;
+    } else if (
+      action.action.component.key === "code" &&
+      action.action.key === "runCode" &&
+      typeof input.value === "string"
```

Comment:
Should this be based on the input type instead of specifically for this action?

Reply:
The input type from the YAML is just `value` so we can't differentiate based on that.